### PR TITLE
CI tests update

### PR DESCRIPTION
**Pull request by alessandrothea**
This PR updates `test-run-sim.sh` (renamed `test-run-sim-eth.sh`), `test-run-sim-udp.sh` and `test-run-sim-slave-counters.sh`, removing outdated bits and introducing logic to wait for Questasim licences.

### CODE REVIEW

**Comment by alessandrothea** [comment]
Run tests please

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@ include:
   - local: '/ci/github-notifications.yml'
   - local: '/ci/templates.yml'
   - local: '/ci/sim.yml'
-  - local: '/ci/publish.yml'
-  - local: '/ci/vivado-checks.yml'
-  - local: '/ci/vivado-builds.yml'
+  # - local: '/ci/publish.yml'
```
Comment (tswilliams) [inline .gitlab-ci.yml:20]: Can you re-enable these?

File: tests/ci/test-run-sim-eth.sh [hunk 2]
```diff
@@ -33,58 +33,57 @@ function print_log_on_error {
   cat ${SIM_LOGFILE}
 }
 
+function wait_for_licence {
+  i=0
+  while [ ! -f ${HAVE_LICENCE_FILE} ]; do 
+    ((i=i+1))
+    m=$(($i%6))
+    if [[ "$m" -eq "0" ]]; then
+      echo "Tik tok $i";
```
Comment (tswilliams) [inline tests/ci/test-run-sim-eth.sh:42]: Could we change this to "Waiting for license" or similar?

File: .gitlab-ci.yml (hunk 1 above)
Comment (alessandrothea) [reply .gitlab-ci.yml]: Yep, just an oversight. Fixed

File: tests/ci/test-run-sim-eth.sh (hunk 2 above)
Comment (alessandrothea) [reply tests/ci/test-run-sim-eth.sh]: And done. But tik tok had its charm

**Comment by alessandrothea** [comment]
Run tests please

**Comment by alessandrothea** [comment]
Run tests please

**Review by tswilliams — APPROVED** [review]

**Review by tswilliams — APPROVED** [review]

**Pull request merged**